test/markup/cpp/function-declarations.expect.txt

```diff
@@ -21,4 +21,3 @@
 
 <span class="hljs-function"><span class="hljs-keyword">void</span> <span class="hljs-title">A</span><span class="hljs-params">()</span>: a(<span class="hljs-number">10</span>) {</span>}
 <span class="hljs-function"><span class="hljs-keyword">explicit</span> <span class="hljs-title">A</span><span class="hljs-params">()</span>: a(<span class="hljs-number">10</span>) {</span>}
```

Comment:
```suggestion
<span class="hljs-function"><span class="hljs-keyword">explicit</span> <span class="hljs-title">A</span><span class="hljs-params">()</span>: a(<span class="hljs-number">10</span>) {</span>}

```

The C++ markup expected file is being touched in this PR by accident it seems. Not sure if you'd want to squash/clean it up somehow.

Reply:
This is just fixing a line break issue, so I don't really care.